qemu/tests/block_resize.py

```diff
@@ -69,35 +70,35 @@ def compare_block_size(session, block_cmd, block_pattern):
                             order_magnitude="B"))
     data_image_filename = storage.get_image_filename(data_image_params,
                                                      data_dir.get_data_dir())
-    data_image_dev = vm.get_block({'file': data_image_filename})
+    data_image_dev = vm.get_block({'file': data_image_filename}).split(" ")[0]
```

Comment:
it looks vm.get_block() will return block device ID directly.  can you set your monitor_type to 'qmp' then try again.